src/mlpack/methods/mean_shift/mean_shift_main.cpp

```diff
@@ -89,7 +92,8 @@ static void mlpackMain()
 
   Timer::Start("clustering");
   Log::Info << "Performing mean shift clustering..." << endl;
-  meanShift.Cluster(dataset, assignments, centroids);
+  meanShift.Cluster(dataset, assignments, centroids,
+   CLI::HasParam("force_convergence"));
```

Comment:
Looks like the spacing is off here---a continued line should be indented four spaces from the previous line.